scout/constants/variants_export.py

```diff
@@ -19,6 +19,9 @@
     "AD Alternative",
 ]
 
+# Used only if app is connected to chanjo report
+MT_COV_STATS_HEADER = ["Mean MT coverage", "Mean autosome coverage", "MT/autosome ratio"]
```

Comment:
Eh, this is semantics but it is rather mean autosome N coverage currently? 😊

Reply:
Yes, I mean to add the chrom, but I still don't know which one to use!